ocfweb/docs/views/officers.py

```diff
@@ -24,12 +25,17 @@ def Term(
     sms: List[Any],
     dgms: Optional[List[Any]] = None,
     dsms: Optional[List[Any]] = None,
+    heads: Optional[List[Tuple[str, List[Any]]]] = None,
```

Comment:
Ideally we'd make the `Any`s more specific. Perhaps this can wait until we can use Python 3.7 data classes.

Reply:
This will get refactored anyways, assuming someone works on https://github.com/ocf/etc/issues/3